rbac/management/querysets.py

```diff
@@ -71,7 +71,9 @@ def get_group_queryset(request):
 
     username = request.query_params.get("username")
     if username:
-        get_principal(username, request.user.account)
+        principal = get_principal(username, request.user.account)
+        if principal.cross_account:
+            return Group.objects.filter(principals__username__iexact=username)
```

Comment:
We just want to return an empty queryset here, correct?

Reply:
Oh! Right, thanks for reminding! Almost forgot about that. I will change it